.travis.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       script: commitlint-travis
     - name: "Code Lint"
       script: npm-run-all clean build:i18n lint
+    - name: "Flow Debug"
```

Comment:
Still needed?

Reply:
Looks like it can be removed. However, if we want to keep this around, can we please fold it into the `Flow Types` job? Something like a console output saying `Running Flow on X files`, with X being `yarn flow ls | wc -l`.

Reply:
Yup. As requested by @alexkrolick

Reply:
We can keep it, but it should be folded into the existing Flow Types task. It doesn't need to be its own task.